test/test_torch.py

```diff
@@ -10465,6 +10465,61 @@ def run_test(tensor_dims, some):
         for tensor_dims, some in product(tensor_dims_list, [True, False]):
             run_test(tensor_dims, some)
 
+    @dtypes(torch.float, torch.double)
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_quantile(self, device, dtype):
+
+        # Generate some random test cases
+        a_sizes = [tuple(np.random.randint(2, 10, size=i)) for i in range(1, 4)]
+        q_values = [tuple(np.random.rand(i)) for i in range(0, 4)]
+
+        # Add corner cases
+        a_sizes.extend([(1,), (1, 1), (1, 2, 1)])
```

Comment:
+ empty tensor (torch.randn(0) will do it) and scalar tensor which I don't think you can test with the current generation method. You could overload the values in `a_sizes` to include a scalar and check in the loop that, if given a scalar, you just create a tensor from it. That is, you want to create a tensor like this:

```
torch.tensor(.75).size()
: torch.Size([])
```

Reply:
Empty input tensor is actually not allowed and results in an exception, but I'll add the scalar input tensor case.